samples/shell/echo/echo.yaml

```diff
@@ -2,6 +2,8 @@ apiVersion: extensions.sk8s.io/v1
 kind: Topic
```

Comment:
should the Topic be removed from this yaml (and this be renamed `echo-function.yaml`) now that you've added `greetings-topic.yaml` on its own?

Reply:
Sure. Note that this doesn't fix the root problem of the topic-controller not adding partitions.

Created https://github.com/markfisher/sk8s/issues/85